aep/general/0003/aep.md.j2

```diff
@@ -33,6 +33,12 @@ Examples of clients include the following:
 - Tools, such as a [Declarative client][]
 - Visual UIs, such as a web application
 
+### AEP Edition
+
+A specific edition of the AEP specification. This is a specific set of AEP
+guidance that will not change, annotated with the year (e.g. aep-2025). See
+[editions](/0300) for more information.
```

Comment:
```suggestion
[editions](/editions) for more information.
```

Didn't we adopt a convention of using slugs rather than numbers for these links?